blazor/datagrid/row-drag-and-drop.md

```diff
@@ -9,7 +9,8 @@ documentation: ug
 
 # Row Drag and Drop in Blazor DataGrid Component
 
-The grid row drag and drop allows you to drag and drop grid rows to another grid or custom component. To enable row drag and drop, set the [AllowRowDragAndDrop](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowRowDragAndDrop) to true. The target component where the grid rows are to be dropped can be set by using the [TargetID](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridRowDropSettings.html#Syncfusion_Blazor_Grids_GridRowDropSettings_TargetID).
+
+The grid row drag and drop feature enables users to move grid rows within the same grid or to another grid or custom component using the drag icon. Users can also achieve this by clicking, dragging, and dropping the rows onto another grid or custom component. To enable row drag and drop, set the [AllowRowDragAndDrop](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowRowDragAndDrop) to true. The target component where the grid rows are to be dropped can be set by using the [TargetID](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridRowDropSettings.html#Syncfusion_Blazor_Grids_GridRowDropSettings_TargetID).
```

Comment:
I have slightly modified the content. Please change it. 
-----
The grid row drag and drop feature enables users to move grid rows within the same grid or to another grid or custom component using the drag icon. Users can utilize the drag icon to move grid rows within the same grid or to another grid or custom component.

Reply:
Hi @Venkat-Ayothi ,

Changed the content as per your suggestion.